assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js

```diff
@@ -88,45 +104,59 @@ const Idea = ( { postEditURL, name, text, topics, buttons } ) => {
 					<p className="googlesitekit-idea-hub__idea--text">{ text }</p>
 				</Cell>
 				<Cell smSize={ 4 } mdSize={ 3 } lgSize={ 3 } className="googlesitekit-idea-hub__idea--actions">
-					{ buttons.includes( IDEA_HUB_BUTTON_DELETE ) && (
-						<Button
-							onClick={ handleDelete }
-							icon={ <DeleteIcon /> }
-							className="googlesitekit-idea-hub__actions--delete"
-							disabled={ isProcessing }
-						/>
-					) }
-
-					{ buttons.includes( IDEA_HUB_BUTTON_PIN ) && (
-						<Button
-							onClick={ handlePin }
-							icon={ <PinIcon /> }
-							className="googlesitekit-idea-hub__actions--pin"
-							disabled={ isProcessing }
-						/>
+					{ [ IDEA_HUB_ACTIVITY_CREATING_DRAFT, IDEA_HUB_ACTIVITY_DRAFT_CREATED ].includes( activity ) && (
+						<div className="googlesitekit-idea-hub__loading-notice">
+							<p>
+								{ activity === IDEA_HUB_ACTIVITY_CREATING_DRAFT
+									? __( 'Creating draft', 'google-site-kit' )
+									: __( 'Draft created', 'google-site-kit' )
+								}
+							</p>
+							<div className="googlesitekit-idea-hub__loading-notice__spinner-wrapper">
+								<CircularProgress size={ 10 } />
+							</div>
```

Comment:
We need to display the progress only for the `IDEA_HUB_ACTIVITY_CREATING_DRAFT` activity.

Reply:
Good point! Re-reading the spec it totally says this :smile: